test/fixtures/basicDefault.vue.js

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue'
+
+export default {
```

Comment:
@Aaron-Pool 
Though not tested, I think this would be turned to `export default {/* ... */}; component.__docgenInfo = // ...` and it'll throw an error, because the loader treat `exportName: 'default'` as SFCs. And if we could fix that, the loader can't handle this case because there is no way to attach `__docgenInfo` to the default export object... 😞 

Adding a warning about the default export on README then create an issue is the safest way, IMO. What do you think?

Reply:
Hmm, I think my changes have have tests passing on both SFC and non-SFC cases? So I'm confused on exactly what you're saying 🤔

Reply:
Oh wait, I think I see what you're saying. Let me dig in a bit more.

Reply:
Sorry for my poor word choices... I mean "`vue-docgen-api` will parse the code successfully, but the loader would append incorrect code, and this will cause a run-time error."

JavaScript code after the process will be looked like this:

```js
import Vue from 'vue'

export default {
  props: {
    /**
     * Foo the number.
     */
    foo: {
      type: Number,
      default: -1
    }
  },
  template: '<span>{{foo}}</span>'
}
;component.__docgenInfo = {/* ... */}
```

Reply:
Right, correct. Is there a reason we couldn't do a simple regex replace of
```
export default {
    ...definition
}
```
with 
```
const component = {
    ...definition
}
export default component;
```
Since the name of the default export, by definition of the esmodule spec, doesn't matter?

Reply:
>  Is there a reason we couldn't do a simple regex replace 

No, we can replace `export default <expression>` (I don't remember why I haven't implemented that transformation.) Now I think this is an existing bug and will fix this in another branch.

Reply:
@pocka I wrote a simple jscodeshift transform before I saw your response. It occurred to me that it might be more reliable than a regex, in case someone uses a weird mangled or minified source code. If you'd like to use it, [you're welcome to it](https://astexplorer.net/#/gist/2dfbbbc37e0f53d31bd6f2a1bedca9c0/latest), but if you have other ideas I understand that 👌

Reply:
Great thanks!